src/app/shared/components/plh-main-header.ts

```diff
@@ -12,7 +12,7 @@ import { AppConfigService } from "../services/app-config/app-config.service";
   template: `<ion-header>
     <ion-toolbar color="primary">
       <ion-buttons slot="start" style="position:absolute">
-        <ion-menu-button *ngIf="showMenuButton"></ion-menu-button>
+        <ion-menu-button *ngIf="showMenuButton" autoHide="false" menu="start"></ion-menu-button>
```

Comment:
The left menu-button auto-hides when split-screen mode enabled as it assumes the menu is now in full display, however as we are actually keeping the left side-menu and only using split-screen for the right-sidebar content prevent this behaviour by setting `autoHide` to false and ensuring it knows to open the left (start) menu